src/wp-includes/block-template-areas.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Block template areas functions.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Registers the navigation areas supported by the current theme. The expected
+ * shape of the argument is:
+ * array(
+ *     'primary'   => 'Primary',
+ *     'secondary' => 'Secondary',
+ *     'tertiary'  => 'Tertiary',
+ * )
+ *
+ * @param array $new_areas Supported navigation areas.
+ */
+function register_navigation_areas( $new_areas ) {
```

Comment:
Should we add a `wp_` prefix to these functions? (I am never sure.)

Reply:
I think it's probably fine without, but a better person to ask might be @peterwilsoncc.

Reply:
I am thinking without the prefix as that matches `register_sidebar`.